client-hc/src/test/java/com/graphhopper/api/Examples.java

```diff
@@ -0,0 +1,100 @@
+package com.graphhopper.api;
+
+import com.graphhopper.GHRequest;
+import com.graphhopper.GHResponse;
+import com.graphhopper.ResponsePath;
+import com.graphhopper.util.Instruction;
+import com.graphhopper.util.InstructionList;
+import com.graphhopper.util.PointList;
+import com.graphhopper.util.shapes.GHPoint;
+import okhttp3.OkHttpClient;
+import org.junit.Ignore;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Locale;
+import java.util.concurrent.TimeUnit;
+
+@Ignore
```

Comment:
For now include it in code for refactoring purposes only. Do not run the tests as we would need an API key etc.